tests/BoreholeFileControllerTest.cs

```diff
@@ -28,9 +29,12 @@ public void TestInitialize()
 
         context = ContextFactory.CreateContext();
 
+        var contextAccessorMock = new Mock<IHttpContextAccessor>(MockBehavior.Strict);
+        contextAccessorMock.Setup(x => x.HttpContext).Returns(new DefaultHttpContext());
+        contextAccessorMock.Object.HttpContext.User = new ClaimsPrincipal(new ClaimsIdentity(new[] { new Claim(ClaimTypes.Name, context.Users.FirstOrDefault().Name) }));
```

Comment:
REQ: Den Usernamen aus `context.Users.FirstOrDefault().Name` in Variable speichern, dann muss dieser nicht wieder bei den einzelnen Tests ausgelesen werden.

Reply:
Ich habe nun das User objekt als variable gespeichert. Hoffe das passt?